src/components/Button/Button.Styled.ts

```diff
@@ -13,12 +13,12 @@ export const StyledButton = styled.button<{
 	border: none;
 	border-radius: 10px;
 	padding: 0.7em;
-	color: ${(props) => props.theme.colors.pureWhite};
-	background-color: ${(props) => props.color || props.theme.colors.PrimaryRed};
+	color: ${({ theme }) => theme.colors.pureWhite};
+	background-color: ${({ theme }) => theme.color || theme.colors.PrimaryRed};
 	cursor: pointer;
 
 	&:hover,
 	&:focus {
-		background-color: ${(props) => props.theme.colors.btnHover};
+		background-color: ${({ theme }) => theme.colors.btnHover};
```

Comment:
Raczej unikałbym nazywania kolorów po tym do czego służą lub gdzie są używane. Jeżeli taki sam kolor będzie Ci potrzebny gdzieś indziej, to zmienisz jego nazwę, stworzysz duplikat lub użyjesz btnHover dla np. backgroundu diva? Generuje to problemy, jeżeli jest to jakiś ciemny zielony przykładowo, to dajesz mu darkGreen i po temacie.